container/utils/__init__.py

```diff
@@ -310,12 +310,9 @@ def get_dependencies_for_role(role_path):
         meta_main_path = os.path.join(role_path, 'meta', 'main.yml')
         if os.path.exists(meta_main_path):
             meta_main = yaml.safe_load(open(meta_main_path))
-            if not meta_main:
-                yield None
-            for dependency in meta_main.get('dependencies', []):
-                yield dependency.get('role', None)
-        else:
```

Comment:
I think this else statement is still needed to account for no meta_main_path?

Reply:
I don't think so: the caller [ignores](https://github.com/pilou-/ansible-container/blob/3ef600e6619273c5f0f6fe5e9f537f9df4634dd9/container/utils/__init__.py#L284) `None` value.